package/MDAnalysis/lib/formats/include/readdcd.h

```diff
@@ -735,13 +735,16 @@ int write_dcdheader(fio_fd fd, const char *remarks, int N,
     fio_write_int32(fd, 0);
   }
   fio_write_int32(fd, 84);
-  fio_write_int32(fd, 164);
+  fio_write_int32(fd, 244);
   fio_write_int32(fd, 3); /* the number of 80 character title strings */
 
   strncpy(title_string, remarks, 240);
+  // Enforce null-termination for long remark strings.
+  // Not a problem for MDAnalysis but maybe for other readers.
+  title_string[239] = '\0';
```

Comment:
I noticed we have a problem with 0 terminating a string if they are longer then 240 characters. This isn't an issue for writing! It might cause issues for reading with external libraries. Our own cython code seems to handle a missing 0 terminator just fine. There is almost no way for us to test this. The cython library converts the char array into proper python string internally. We never get to see the raw char buffer in python. 

I can again to a test of the expected byte. But nothing more.

Reply:
I adjusted the tests and docs for this change. This way we can be sure that any DCD written by MDAnalysis doesn't trip over another C library reading DCD.